ui/src/app/base/hooks.js

```diff
@@ -79,7 +79,10 @@ export const useFormikErrors = errors => {
  * @param {Object} allowAllEmpty - Whether all fields are allowed to be empty.
  * @returns {Boolean} Form is disabled.
  */
-export const useFormikFormDisabled = allowAllEmpty => {
+export const useFormikFormDisabled = ({
+  allowAllEmpty = false,
+  allowUnchanged = false
```

Comment:
`allowUnchanged` needs to be documented above

Reply:
thanks @squidsoup for keeping me honest :P